src/app/ansyn/ansyn.component.ts

```diff
@@ -84,5 +84,7 @@ export class AnsynComponent implements OnInit{
 		this.mapsContainer.imageriesContainer.nativeElement.classList.toggle('edit-mode');
 	}
 
-
+	backToWorldView($event) {
+		this.store.dispatch(new BackToWorldAction({ mapId: undefined}));
```

Comment:
change to this.store.dispatch(new BackToWorldAction());
  
            
  Write